spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -225,1255 +229,1336 @@
 
   describe '#perform' do
     describe 'IPP enabled' do
-      describe 'Proofed without secondary id' do
-        let!(:pending_enrollments) do
-          ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
-            create(
-              :in_person_enrollment,
-              :pending,
-              :with_notification_phone_configuration,
-              issuer: 'http://localhost:3000',
-              selected_location_details: { name: name },
-            )
+      describe 'In-Person Proofing' do
+        describe 'Proofed without secondary id' do
+          let!(:pending_enrollments) do
+            ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
+              create(
+                :in_person_enrollment,
+                :pending,
+                :with_notification_phone_configuration,
+                issuer: 'http://localhost:3000',
+                selected_location_details: { name: name },
+                sponsor_id: usps_ipp_sponsor_id,
```

Comment:
Non-blocking feedback: 

Since we're moving toward not having any in-person enrollments with a nil sponsor_id, I think it would be smart to update the `in_person_enrollment` factory to always have a sponsor_id.  We could have the factory default to using the `usps_ipp_sponsor_id` and then have a trait for `enhanced_ipp`, which would use `usps_eipp_sponsor_id` as the sponsor_id.

Thoughts?

Reply:
Thank you for bringing this up!  IMO, it's an anti-pattern to leave a one-off rake task and its supporting tests in a code base after the one-time task has been run.  I should have already deleted `lib/tasks/backfill_sponsor_id.rake` and `lib/tasks/backfill_sponsor_id_rake_spec.rb`.  I created PR #10929 to do so.

(For context, there is already precedent for deleting one-time rake tasks - check out PR #9769.)

Reply:
Thanks for approving PR #10929.  I just merged it.